Library/Formula/symlinks.rb

```diff
@@ -0,0 +1,23 @@
+class Symlinks < Formula
+  desc "scan/change symbolic links"
+  homepage "http://www.ibiblio.org/pub/Linux/utils/file/symlinks.lsm"
```

Comment:
Are you questioning whether `-isysroot` does what I claim it does or are you questioning whether `superenv` does what I claim it does in absence of the CLT?